packages/nimble-components/src/select/specs/filerable-select-hld.md

```diff
@@ -0,0 +1,78 @@
+# Filterable Select
+
+## Problem Statement
+
+Our clients have a need of a filterable dropdown component that does not require arbitrary text as a value, but only values available in the dropdown. The filtering will have the following behaviors:
```

Comment:
I find the wording of _that does not require arbitrary text as a value_ a little confusing/misleading. Really, it's not that they don't require arbitrary text, it's that they don't allow arbitrary text, right? What would you think about replacing the word _require_ with _allow_?

```suggestion
Our clients have a need of a filterable dropdown component that does not allow arbitrary text as a value, but only values available in the dropdown. The filtering will have the following behaviors:
```